src/libraries/Common/src/Interop/Windows/Kernel32/Interop.GetFileInformationByHandle.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Win32.SafeHandles;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Kernel32
+    {
+        [DllImport(Libraries.Kernel32, SetLastError = true, ExactSpelling = true)]
```

Comment:
@danmoseley In tests I get a error here (`Fatal error. 0xC0000005`). I think a segfault or so...

Do you know if here is something wrong with the implementation?
```cpp
BOOL GetFileInformationByHandle(
  [in]  HANDLE                       hFile,
  [out] LPBY_HANDLE_FILE_INFORMATION lpFileInformation
);
```

https://docs.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-getfileinformationbyhandle

Reply:
@Jozkee will have to help with that one.

Reply:
@danmoseley @Jozkee okay :smile:

Reply:
@danmoseley @Jozkee I have fixed the problem myself in the meantime. But now I get an UnauthorizedAccessException. 

`System.UnauthorizedAccessException : Access to the path 'C:\h\w\AE80098F\t\File_GetSetTimes_ru3hdau2.2wl\SetLastWriteTimeTicks_SafeFileHandle_254_3275ead0' is denied.`

Do you have any suggestions to solve this?

Reply:
You are using `File.OpenHandle` without specifying `FileAccess`, that means that the handle is open with read permissions by default, and you also need write permission.